### PR TITLE
Remove makefile

### PR DESCRIPTION
**Pull request by uri-weisman**
### Summary of your changes
Remove makefile functionality.

### Related Issues
- Related: https://github.com/elastic/cloudbeat/issues/466

### Checklist
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary README/documentation (if appropriate)

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ repos:
         files: ^tests/
 
   - repo: https://github.com/pycqa/pylint
-    rev: v2.15.7
+    rev: v3.0.0a6
```
Comment (orestisfl) [inline .pre-commit-config.yaml:40]: Let's not use an alpha release

**Review by orestisfl — APPROVED** [review]

**Comment by uri-weisman** [comment]
I've performed a tarball comparison between `main` and my branch.
Seems that there's no change in cloudbeat's binary, only with the CSP policies bundle's location inside the archive (which makes sense I guess).

[tarball diff report](https://drive.google.com/file/d/1Bu784oPYWJZ1EVAEMwvneDwhiuSDhD08/view?usp=sharing)

**Comment by uri-weisman** [comment]
> Can you please explain?

@oren-zohar, in the report I've attached, we can notice that no file was modified besides renaming the folder of the CSP-policies bundle inside the tarball.
Screenshot 2023-05-31 at 17 03 59

**Comment by uri-weisman** [comment]
@oren-zohar, that is correct, the bundle id is not deterministic.
I've run the `package` cmd twice on the main branch and recieved two different ids.

**Review by oren-zohar — APPROVED** [review]

**Pull request merged**